### DISCUSSION TITLE
네이버 커머스 API센터의 고객 문의 조회 요청 관련 문의

### DISCUSSION THREAD

**Discussion by minsuck1000** [commerce-api-naver/commerce-solutions#72] [묻고 답하기]
안녕하세요
TWC 박민석 입니다.

네이버 커머스 API센터의 고객 문의 조회를 호출하여 데어터 적재 및 가공 처리를 하고 있습니다.
고객의 문의 조회는 잘 이용하고 있는 상태이며, 문의가 삭제 되었을때 호출 하는 API가 보이지 않아 확인 요청 드립니다.
고객 문의가 삭제 된 경우 API가 존재하면 URL을 부탁드리며, 존재하지 않는다면 개발 계획은 있는지 알려 주세요

감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

현재 삭제된 고객 문의와 관련한 정보는 API 채널로 제공하지 않고 있습니다.
삭제된 고객 문의는 구매자 분께서 문의 내용의 철회와 삭제 의사를 표명해주신 데이터이기에 현 시점에서는 관련 정보를 API로 제공할 계획을 갖고있진 않습니다.
